builder/builder-next/worker/containerdworker.go

```diff
@@ -28,7 +28,7 @@ func NewContainerdWorker(ctx context.Context, wo base.WorkerOpt) (*ContainerdWor
 // Exporter returns exporter by name
 func (w *ContainerdWorker) Exporter(name string, sm *session.Manager) (exporter.Exporter, error) {
 	switch name {
-	case mobyexporter.Moby:
+	case mobyexporter.Moby, client.ExporterImage:
```

Comment:
This will force unpack (and other things we force with the wrapper) to any build request with the image exporter, I’m not sure we want to do that

Reply:
What would be the the use case for using plain `image` with moby?

Reply:
That's kind of my perspective too - `moby` exporter is just an implementation detail that allows us to override some options that make more sense in our context. It shouldn't really be visible on the outside and from the outside it would still be the `image` exporter.

Reply:
My worry is that, with this change, the `image` exporter doesn't work as documented, since we force things.

And if moby does implement the `image` exporter then we can end up being able to build an image (inside our own `moby` containerd namespace) that's not visible by docker (because it's not unpacked for example), or can very well break docker if we don't handle each and every corner case.

Reply:
We've established that they should (in large part) be equivalent after talking with @tonistiigi; the issue is more where the should should live in the long-term (https://github.com/moby/moby/pull/45666#pullrequestreview-1456184882).

For now, it's probably best to create an issue on BuildKit to move the wrapper functionality to that repo, and put a `TODO:` here linking to it.

Reply:
> In my mind, moby is the exporter that will get you an image inside moby

`type=docker` exporter is what gets image inside `docker`, even if building inside container/k8s etc. `type=moby` is not even documented anywhere https://docs.docker.com/engine/reference/commandline/buildx_build/#output . It's an internal switch that `buildx` does because non-containerd moby is incapable of using the image exporter directly.